docs/open-source/action-phrase-triggers.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Action Triggers'
+description: 'Activate actions with function calls or specific phrases'
+---
+## What are function calls?
+[Function calls](https://platform.openai.com/docs/guides/function-calling) allow modern Large Language Models (LLMs) like ChatGPT to perform tasks outside of text generation. 
+For example, if I create a math assistant with ChatGPT, I might make a "multiply" function that ChatGPT can call to multiply two numbers together.
+
+If you add an action to a vocode agent, it is automatically added to each ChatGPT query. If the function call is outputted by ChatGPT, the corresponding action is triggered automatically.
```

Comment:
change to something like "If you add an action to a Vocode agent, the corresponding function calling schema is added to each ChatGPT query"